### PR TITLE
Lwr default tarcan

### PR DESCRIPTION
**Pull request by tarcang**
### What does this PR do?
@W-12226848@

- Right now I set the "B2B Commerce (LWR)" as default for B2B store creation. 
- The B2BAuraTemplateEnabled permission is added to the default scratch definition files (this was already by @joeluong-sfcc) and overwritten in .commerce folder.

### CODE REVIEW

File: config/both-project-scratch-def.json [hunk 1]
```diff
@@ -1,38 +1,38 @@
...
+            "enableOptionalPricebook": true
+        },
+        "commerceSettings": {
+            "commerceEnabled": true
+        },
+        "languageSettings": {
+            "enableTranslationWorkbench": true
+        }
     }
-  }
```
Comment (tarcang) [inline config/both-project-scratch-def.json:37]: I think the linter in husky is changing this. There is no change in the file.

File: config/b2b-project-scratch-def.json [hunk 2]
```diff
@@ -1,27 +1,27 @@
...
+        "orderSettings": {
+            "enableOrders": true,
+            "enableEnhancedCommerceOrders": true,
+            "enableOptionalPricebook": true
+        },
+        "commerceSettings": {
+            "commerceEnabled": true
+        }
     }
-  }
```
Comment (tarcang) [inline config/b2b-project-scratch-def.json:26]: The linter change again.

File: config/both-project-scratch-def.json (hunk 1 above)
Comment (unandyala) [reply config/both-project-scratch-def.json:37]: please revert the linter changes. I think we all should have the same linter configured

**Review by unandyala — APPROVED** [review]

**Review by joeluong-sfcc — APPROVED** [review]

**Comment by tarcang** [comment]
Carried all the changes to https://github.com/forcedotcom/commerce-on-lightning/pull/153

**Pull request closed without merging**